src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -92,7 +99,8 @@
     interface ComputedFromDownloadResults {
         spectrum: DownloadResultsDependency,
         coarseOutput: DownloadResultsDependency,
-        summary: DownloadResultsDependency
+        summary: DownloadResultsDependency,
+        comparison: DownloadResultsDependency
```

Comment:
So I'm guessing at this point we are intentionally downloading the summary report in place of the comparison report?

Reply:
Yes it’s intentional